modules/nlu/src/backend/typings.ts

```diff
@@ -194,3 +194,10 @@ type SlotDefinition = Readonly<{
   name: string
   entities: string[]
 }>
+
+export class TrainingCanceledError extends Error {
```

Comment:
as I said, you could not know this, but erros of Type TrainingCancelError should be catched inside the training pipeline so you don't need to export it in `typings.ts`

altough to be fair, `typings.ts` was the correct place to export such a type!